backend/de.metas.acct.base/src/main/sql/postgresql/system/43-de.metas.acct/5691075_sys_gh_15528_add_translations_for_AllocateToFEC_process.sql

```diff
@@ -0,0 +1,8 @@
+update ad_element set name='Foreign Exchange Contract',printname='Foreign Exchange Contract'
+                  FROM ad_element where ad_ELEMENT_id = 581935;
+
+update ad_element set name='Allocated Amount',printname='Allocated Amt'
+                  FROM ad_element where ad_ELEMENT_id = 2677;
+
+update ad_element set name='Remaining FEC',printname='Remaining FEC'
```

Comment:
why not generate the scripts?

asking beucase in this way it looks like ad_element_trl is not updated.... nor the trl pushed to depending entities.

Reply:
long story short: the generated scripts are here: backend/de.metas.acct.base/src/main/sql/postgresql/system/43-de.metas.acct/5691070_sys_gh_15528_add_translations_for_AllocateToFEC_process.sql ; but bc of the base language on the local instance (DE) vs abeamuat (EN) the german text is displayed even if `EN` language is set bc I assume the values are coming from the ad_element instead of ad_element_trl when the user has base language set

Reply:
@cp-ps 
that's not good. pls next time make sure you have a db similar with customer's one. pls ask in chat if needed.

@cp-ps
I thought that issue is already fixed when running migration scripts because the migration scripts runner is calling
`after_migration_sync_translations()` 
which calls 
`update_TRL_Tables_On_AD_Element_TRL_Update();`
which calls
`update_ad_element_on_ad_element_trl_update(null, null);`
which shall update all AD_Element(s) from AD_Element_Trl(s).

Isn't that the case?

NOTE: I would really like to rely on technologies we have instead of applying workarounds.